File: InsuranceExamGAP-ORM/Models/Policy.cs
using System;

namespace InsuranceExamGAP_ORM.Core.Models
{
    public class Policy
    {

        public int? PolicyId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int PolicyTypeId { get; set; }
        public PolicyType PolicyType { get; set; }
        public DateTime PolicyDateBegin { get; set; }
        public int PolicyMonthsPeriod { get; set; }
        public double PolicyCost { get; set; }
        public int RiskTypeId { get; set; }
        public RiskType RiskType { get; set; }

    }
}

File: InsuranceExamGAP/ClientApp/src/app/_models/policy.ts
import { PolicyType, RiskType, } from '../_models';
export class Policy {
  policyId: number;
  name: string;
  description: string;
  policyTypeId: number;
  policyDateBegin: string;
  policyMonthsPeriod: number;
  policyCost: number;
  riskTypeId: number;
  riskType: RiskType;
  policyType: PolicyType;
}
